### PR TITLE
Users Tab Template

### PR DESCRIPTION
**Pull request by j-jasik**
The basic layout of the Users tab.

### CODE REVIEW

File: sciencelabs/__init__.py [hunk 1]
```diff
@@ -16,13 +16,15 @@
 sentry = Sentry(app, dsn=app.config['SENTRY_URL'], logging=True, level=logging.INFO)
 
 from sciencelabs.views import View
```
Comment (c-schwarze) [inline sciencelabs/__init__.py]: Can we rename View for the base one to be called something different? BaseView, IndexView, etc.

Comment (bam95899) [reply sciencelabs/__init__.py]: Changed to IndexView

Comment (c-schwarze) [reply sciencelabs/__init__.py]: FrontView is another option.

What all is under IndexView? Is it anything other than the homepage? If so, it should be something like BaseView/FrontView. 

(my bad, i didn't think about IndexView enough)

Comment (bam95899) [reply sciencelabs/__init__.py]: IndexView creates the homepage using sciencelabs_base.html by extending it.

**Pull request merged**